src/main/java/org/folio/oaipmh/WebClientProvider.java

```diff
@@ -34,7 +34,8 @@ public static void init(Vertx v) {
     webClient = WebClient.create(vertx);
     WebClientOptions options = new WebClientOptions()
       .setKeepAliveTimeout(KEEP_ALIVE_SEC)
-      .setConnectTimeout(DOWNLOAD_INSTANCE_CONNECTION_TIMEOUT_MS);
+      .setConnectTimeout(DOWNLOAD_INSTANCE_CONNECTION_TIMEOUT_MS)
+      .setMaxChunkSize(256).setReceiveBufferSize(10 * 1024);
```

Comment:
Is it make sense to set these values as constants?

Reply:
Done